create_test.go

```diff
@@ -1074,43 +1085,50 @@ func TestCopyParentCPUSetFail(t *testing.T) {
 }
 
 func TestCopyParentCPUSetSuccessful(t *testing.T) {
-	if os.Geteuid() != 0 {
-		// Create cgroup directory must be root.
-		t.Skip(testDisabledNeedRoot)
-	}
-
 	assert := assert.New(t)
 
-	cgroupsRootPath := findCgroupRoot()
-	if cgroupsRootPath == "" {
-		t.Skip(testDisabledNeedCgroup)
+	cgroupsPath, err := ioutil.TempDir(testDir, "cgroups-path-")
+	if err != nil {
```

Comment:
All the `if err!= nil` tests can be replaced with the simpler and shorter `assert.NoErrror()` syntax. For example:

```
assert.NoErrror(err, "Could not create temporary cgroups directory")
```

Reply:
I follow the style from this function
```
func TestCgroupsFilesNonEmptyCgroupsPathSuccessful(t *testing.T) {
	cgroupsPath, err := ioutil.TempDir(testDir, "cgroups-path-")
	if err != nil {
		t.Fatalf("Could not create temporary cgroups directory: %s", err)
	}
       ...
```

and I found that this code style exist in another function `TestCreatePIDFileSuccessful`
```
func TestCreatePIDFileSuccessful(t *testing.T) {
      pidDirPath, err := ioutil.TempDir(testDir, "pid-path-")
      if err != nil {
          t.Fatalf("Could not create temporary PID directory: %s", err)
      }
 ... 

```
I wonder if we can use `assert.NoError(err)` to simplify code for these functions, and make style  consistent

Reply:
Yes, there is still test code that uses the old `if err != nil` style, but we are trying to make all new test code use `assert.*` as it is simpler, easier to read and shorter ;) When we get a chance, we'll update the older test code to use `assert.*` too.

As I say, whichever style you chose isn't going to block this code landing, but it's just an "FYI" for future.

Reply:
@jodh-intel  , I update the code, use `assert.*` instead of `if err != nil`